### ISSUE TITLE
Missing values

### ISSUE THREAD

**Issue by MotoRZR** [vecxoz/vecstack#14]
Hi great function! What if I data with missing values and I want to leave them as missing for the purpose of an XGBClassifier in an ensemble but I also want to include an Sklearn classifier that requires missing values to be filled e.g., Random Forest.  So basically my training data would be different for different models in the ensemble.

Another related example would be encoding of categorical variables.  For LGBMClassifier I may  want to label encode vs one hot encode for XGBoost so the training set would have different dimensions for each classifier in this example

**Comment by vecxoz** [comment]
Hi! Thanks!

**About different data for different models**

  When you call `stacking` function or `StackingTransformer.fit` each model will get the **same data**. User has no control over data inside. To pass different data you need to make different call. 

So basic approach is to collect models which you want to train on label encoding in 1st group and run `stacking` function, then collect models which you want to train on one-hot encoding in 2nd group and run `stacking` function again. Finally you just combine results from this two runs using `np.hstack`. `StackingTransformer` in combination with appropriate preprocessing steps can be used inside `FeatureUnion` for this purpose.

**About missing values** 

`StackingTransformer` does not allow `np.nan` in data at all. This is required by sklearn compliance test. `stacking` function allows `np.nan`.

But for your task you actually don’t need `np.nan`. You can train ensemble of `XGBClassifier` and `RandomForestClassifier` in a single run using the following approach:

Fill `np.nan` with some unique numeric value that does not appear anywhere else (like `-9999`). Initialize `XGBClassifier` with [`missing`](https://xgboost.readthedocs.io/en/latest/python/python_api.html#module-xgboost.sklearn) parameter like so: `XGBClassifier(missing=-9999)`. Now `XGBClassifier` will treat `-9999` like `np.nan`, and `RandomForestClassifier` will treat it like usual number.
